package.json

```diff
@@ -303,12 +309,15 @@
     "postinstall-postinstall": "^2.1.0",
     "progress-bar-webpack-plugin": "^2.1.0",
     "raw-loader": "^4.0.1",
-    "react": "^16.13.1",
+    "react": "^16.14.0",
```

Comment:
Does this PR require React upgrade?

Reply:
Nope.

Reply:
Should we revert? Not that many changes in that release: https://github.com/facebook/react/blob/master/CHANGELOG.md#16140-october-14-2020

Reply:
Why it's even matter? :) It is not a major release update.